core/templates/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -124,6 +124,51 @@ <h3>Update username</h3>
     </form>
   </div>
 </md-card>
+
+<md-card class="oppia-admin-page-card oppia-page-card oppia-long-text">
+  <h2>Wipeout</h2>
+  <h3>Number of pending deletion requests</h3>
+  <div class="oppia-query-form">
+    <form class="form"
+          ng-submit="$ctrl.getNumberOfPendingDeletionRequestModels()">
+      <div class="form-group row">
+        <input type="submit" value="Get number of PendingDeletionRequestModels">
+      </div>
+    </form>
+  </div>
+  <h3>Delete account</h3>
+  <div class="oppia-query-form">
+    <form class="form"
+          ng-submit="$ctrl.deleteAccount()">
+      <div class="form-group row">
+         (the user needs to request the deletion before this can be run)
```

Comment:
Is there a reason why we don't just do it automatically? (Just checking.)

What happens if the user has not yet requested the deletion?

Reply:
I will be done automatically in most cases (by cron).

It will return an exception that there is no pending deletion request for this user.